qtranslate_core.php

```diff
@@ -1466,7 +1466,7 @@ function qtranxf_use_content($lang, $content, $available_langs, $show_available=
 	$alt_content = $content[$alt_lang];
 	$alt_lang_is_default = $alt_lang == $q_config['default_language'];
 
-	if(!$show_available){
+	if(!$show_available || !$q_config['show_alternative_content_message']){
```

Comment:
Is not it a bit strange to override the value of argument $show_available supplied in the function call? Seems like it would be better to change the value of argument when the function is called. That argument is already doing the same thing as the option, and now you have to remember that providing certain argument is not enough? I understand, it minimizes the amount of changes to achieve your goal now, but if you were writing this code from the beginning, you would not use such logic, would you?

I was worried before that the list of available languages is only needed in post content, and the argument $show_available is set to 'true' in a very few places, namely, I found two:

https://github.com/qTranslate-Team/qtranslate-x/blob/master/qtranslate_frontend.php#L497

https://github.com/qTranslate-Team/qtranslate-x/blob/master/qtranslate_frontend.php#L726

which makes me think that having a separate function for this case would make more sense, instead of setting additional useless argument to 'false' everywhere else.

Anyway, I guess, that would be too much to ask you to do, I may arrange it later. This note is rather a reminder for myself. I'll merge this code a bit later, since the qtx_configuration.php is severely modified now and not checked in. I will merge this one after I check in. Thanks a lot, Michele, for all your new enhancements, much appreciated :+1: